tools/bindsnoop.py

```diff
@@ -418,9 +432,45 @@ def opts2str(bitfield):
         bit *= 2
     return str_options.encode()
 
+def opts2array(bitfield):
+    options = []
+    bit = 1
+    for opt in "FTNRr":
+        if bitfield & bit:
+            options.append(opt)
+        bit *= 2
+    return options
+
 if args.containersmap:
     containers_map = ContainersMap(args.containersmap)
 
+def print_json(event, inetType):
+    global start_ts
+    # always print timestamp
+    if start_ts == 0:
+            start_ts = event.ts_us
+    eventJ = {
+        "time": ((float(event.ts_us) - start_ts) / 1000000),
```

Comment:
For another PR...

I know the same logic was in the initial code, but I wonder if we should use RFC3339 (`2012-04-23T18:25:43.511Z`).

```
import datetime
datetime.datetime.now().isoformat()
```

But I don't know how to convert the clock from bpf_ktime_get_ns() / CLOCK_MONOTONIC into the wall clock. Same problem with bpf_ktime_get_boot_ns() / CLOCK_BOOTTIME...

Reply:
I agree. This shouldn't be a relative time for the inspektor gadget use case. Let me open an issue to track this down.